lib/helpers/reportback.js

```diff
@@ -0,0 +1,65 @@
+'use strict';
+
+const config = require('../../config/lib/helpers/reportback');
+
+/**
+ * @param {string} message
+ * @return {boolean}
+ */
+function hasLetters(message) {
+  return RegExp(/[a-zA-Z]/g).test(message);
+}
+
+/**
+ * TODO: Make this much better!
+ * @see https://github.com/DoSomething/gambit/issues/608
+ *
+ * Checks if given input string contains a valid Reportback quantity.
+ * @param {string} input
+ * @return {boolean}
+ */
+function validateQuantity(quantity) {
+  // Letters are invalid
+  if (hasLetters(quantity)) return false;
+
+  const numericQuantity = Number(quantity);
+
+  if (numericQuantity && numericQuantity === parseInt(numericQuantity, 10)) {
+    return true;
+  }
+
+  return false;
+}
+
+/**
+ * Checks if given input string contains a valid Reportback text field.
+ * @param {string} input
+ * @return {boolean}
+ */
+function validateText(input) {
+  return !!(input && input.trim().length > config.minTextLength && hasLetters(input));
+}
+
+/**
+ * Trims given input with an ellipsis if its length is greater than 255|maxTextLength characters.
+ * @param {string} input
+ * @return {string}
+ */
+function trimText(input) {
+  const maxLength = config.maxTextLength;
+  const ellipsis = config.ellipsis;
+  const keepLength = maxLength - ellipsis.length;
+  let text = input.trim();
+
+  if (text.length > maxLength) {
+    text = `${text.substring(0, keepLength)}${ellipsis}`;
```

Comment:
@DFurnes @sbsmith86 -- We're currently trimming Caption and Why Participated if they have lenght greater than 255 characters - due to posting to the Ashes endpoint and #881. Is this still required?

Reply:
@aaronschachter `caption` is still limited to 255 but we can update that. `why_participated` is a `TEXT`field so it shouldn't be a concern.

Reply:
I think it might still make sense to restrict length on captions (since we have to then show them in the gallery w/ a limited amount of space). We validate [4-80 characters](https://github.com/DoSomething/phoenix-next/blob/ca5ef704046b93e233449d7cd11a5c9acf3d8544/app/Http/Controllers/ReportbackController.php#L44) in Phoenix Next, for example.

Reply:
Hmm.. so should we trim text past 60(?) characters for captions? Right now its trimming to 255 characters. @DFurnes ? I'm going to remove the trimming on the why participated field based on your feedback @sbsmith86 @DFurnes.

Reply:
I don't think it's an absolute requirement, but it does make it easier to display captions in the UI without having to truncate text. IMO, it'd make sense to enforce the same 80 character limit everywhere.